standards.md

```diff
@@ -0,0 +1,315 @@
+
+<p>&nbsp;</p>
+
+The coordinates keys specify ranges of acceptable time and spatial resolution:
+| Key  | Definition |
+|------|------------|
+| min_time_range<br>max_time_range | (optional) CMOR3.5 or UDUnits-compatible strings specifying the minimum and/or maximum length of analysis period the DEM requires. For example, a DEM that uses monthly data may need to set a minimum time range in order to guarantee it has enough data to compute statistically meaningful results. If this is not specified, any time duration is allowed. |
+| min_resolution<br>max_resolution  | (optional) Should be one of the CMIP6 nominal_resolution values, with "nominal resolution" defined as in the [CMIP6 Global Attributes document](http://goo.gl/v1drZl). These specify the minimum and/or maximum spatial resolution the DEM’s data should have for the DEM to produce meaningful results. If this is not specified, any resolution is allowed. |
+
+<p>&nbsp;</p>
+
+### Contents file <a name="contents"></a>
+
+If there are multiple configurations within the DEM, there must be a JSON file named “contents.json” that contains the module name and a list of the settings files within the DEM.
+
+The required base level keys of the settings file are as follows:
+| Key  | Definition |
+|------|------------|
+| module | Provides the DEM name. |
+| contents | List of all DEM settings file names (relative to DEM directory). |
+
+<p>&nbsp;</p>
+
+The following module keys are required:
+| Key  | Definition |
+|------|------------|
+| name | Short name of DEM |
+| long_name | A human readable name of module |
+
+<p>&nbsp;</p>
+
+## Runtime Environment <a name="runtimeenvironment"></a>
+
+This section describes the runtime environment, as initialized by the framework. When executed through the CMEC/MDTF interface, the DEMs may assume that all conditions below are met.
+
+All executables and third-party libraries requested in the settings file will be available on the $PATH seen by the DEM.  
+
+The following environment variables will be set:
+
+| Key  | Definition |
+|------|------------|
+| CMEC_CODE_DIR <br>MDTF_POD_CODE_DIR| The directory where the DEM’s code can be found |
+| CMEC_OBS_DATA<br>MDTF_POD_OBS_DATA| The directory where the relevant observational data can be found. |
+| CMEC_MODEL_DATA<br>MDTF_MODEL_DATA | The directory containing the model input data.  If autocurated metadata has been generated, it can be found in the file autocurator.json in the root directory. |
+| CMEC_WK_DIR<br>MDTF_POD_WK_DIR | The working directory for output from the DEM. |
+| MDTF_CASENAME<br>MDTF_FIRSTYR<br>MDTF_LASTYR | (optional) Variables used by MDTF PODs for labeling plots. |
+| CMEC_DEBUG<br>MDTF_DEBUG | (optional) If set to 1 then the DEM should output more verbose logging information to help with debugging. |
+| CMEC_SAVE_NC<br>MDTF_SAVE_NC | (optional) If set to 0 then the DEM should skip any steps where raw output data is saved in netCDF format. |
+
```

Comment:
I recall a proposal to rename all of these variables `EMDS_*`  -- a heads up to @wrongkindofdoctor and @aradhakrishnanGFDL

Reply:
@jkrasting I opened an issue for this proposal (https://github.com/Earth-System-Diagnostics-Standards/EMDS/issues/4). Not sure if it would be ok to make those changes now in the standards doc, or if we want to change over the variables in the software first?